IOSExercises/Models/Person+CoreDataClass.h

```diff
@@ -0,0 +1,29 @@
+//
+//  Person+CoreDataClass.h
+//  IOSExercises
+//
+//  Created by Arnold Joseph Caesar Esteban on 12/9/20.
+//  Copyright © 2020 Arnold Joseph Caesar Esteban. All rights reserved.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import <CoreData/CoreData.h>
+#import "EXRColor.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Person : NSManagedObject
+
++ (Person *)createEmptyPersonWithContext:(NSManagedObjectContext *)aContext;
+
++ (NSArray<Person *> *)fetchAllPersonsWithContext:(NSManagedObjectContext *) aContext;
```

Comment:
```suggestion
+ (NSArray<Person *> *)personsWithContext:(NSManagedObjectContext *)context;
```

No need to include the action "fetch" in the name